docs/config.yml

```diff
@@ -39,9 +24,18 @@ html:
     hypothesis: false
     utterances: false
 sphinx:
-  extra_extensions: []
+  extra_extensions: 
+  - sphinx_sitemap
+  - sphinx.ext.napoleon
+  - sphinx.ext.todo
+  - sphinx.ext.autodoc
+  - sphinx.ext.viewcode
   local_extensions: {}
   config: 
+    html_baseurl: "https://schemata.readthedocs.io/en/latest/"
+    napoleon_numpy_docstring: True
+    autodoc_inherit_docstrings: True
+    autodoc_typehints: "none"
```

Comment:
perhaps try this to get better cross-links, for at least stdlib:

```yaml
config:
  intersphinx_mapping =
    python: ["https://docs.python.org/3", null],
    traitlets: ["https://traitlets.readthedocs.io/en/stable/", null]
```

also would be interested to see what happens vs https://pypi.org/project/sphinx-autodoc-typehints/